rpm/default/arangodb3.spec.in

```diff
@@ -85,11 +85,20 @@ mkdir -p %{buildroot}%{_appdir}
 mkdir -p %{buildroot}%{_logdir}
 mkdir -p %{buildroot}%{_piddir}
 cp -a $INNERWORKDIR/ArangoDB/build/install/* %{buildroot}
+rm -f %{buildroot}/README
 # Move go executables out of the way to prevent debug info from being stripped:
-mv %{buildroot}/usr/bin/arangodb /tmp
+mkdir /tmp/sbin /tmp/bin
+# arangodb
+mv %{buildroot}/usr/bin/arangodb /tmp/bin
+# arangosync
 if [ -x %{buildroot}/usr/sbin/arangosync ] ; then
-  mv %{buildroot}/usr/sbin/arangosync /tmp
+  mv %{buildroot}/usr/sbin/arangosync /tmp/sbin
 fi
```

Comment:
Is it safe to run this when we have multiple build processes running?
Can they share the same `/tmp` directory? If so, then this would need some form of synchronization in order to be free from races.

Reply:
That is `/tmp` inside docker container which is isolated (per build package function) in case of Linux. However, `/work` is limited to a single oskar instance. Therefore 2 DEBs or 2 RPMs can't be built at the same time but we can improve `buildPackage` function to run `TAR.GZ`, `DEB` and `RPM` packages creation in background.